src/components/views/avatars/DecoratedRoomAvatar.tsx

```diff
@@ -159,6 +159,8 @@ export default class DecoratedRoomAvatar extends React.PureComponent<IProps, ISt
     }
 
     private calculateIcon(): Icon {
+        if (this.props.room.isSpaceRoom()) return Icon.None;
```

Comment:
I think in other places we show a plain old `<RoomAvatar />` for spaces, instead of a Decorated one, can we be consistent (in either direction) please